CMakeLists.txt

```diff
@@ -77,13 +77,15 @@ target_link_libraries(multiprocess PRIVATE CapnProto::kj)
 target_link_libraries(multiprocess PRIVATE CapnProto::kj-async)
 set_target_properties(multiprocess PROPERTIES
     PUBLIC_HEADER "${MP_PUBLIC_HEADERS}")
-install(TARGETS multiprocess EXPORT Multiprocess
+install(TARGETS multiprocess EXPORT libmultiprocess-lib
   ARCHIVE DESTINATION lib COMPONENT lib
   PUBLIC_HEADER DESTINATION include/mp COMPONENT lib)
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/pkgconfig/libmultiprocess.pc"
   DESTINATION "lib/pkgconfig" COMPONENT lib)
+install(EXPORT libmultiprocess-lib DESTINATION lib/cmake/Multiprocess)
```

Comment:
> Could you change `lib/cmake/Multiprocess` to `${CMAKE_INSTALL_LIBDIR}/cmake`?

[Done](https://github.com/chaincodelabs/libmultiprocess/pull/79#issuecomment-1406560767) in a new "Use `GNUInstallDirs` module" commit.

> I know `Multiprocess` subdirectory was used before this PR, but I think that directory name is a little misleading because it doesn't match the project name. Also, there is not really a point in creating a subdirectory since this is just exporting targets, not a full cmake package (https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html#creating-packages, https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure)

Thanks! [Updated](https://github.com/chaincodelabs/libmultiprocess/pull/79#issuecomment-1406560767).